components/Login/LoginForm.js

```diff
@@ -15,15 +15,32 @@ export default class LoginForm extends Component {
     this.state = {
       user: null,
       email: null,
-      password: null
+      password: null,
+      error: null,
+      response: null
     }
 
     this._postForm = this._postForm.bind(this)
   }
 
   _postForm () {
-    this.setState({
-      user: !this.state.user
+    const url = 'https://neomad.org/api/login/'
+    const data = {
+      email: this.state.email,
+      password: this.state.password
+    }
+
+    fetch(url, {
+      method: 'POST',
+      body: JSON.stringify(data),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      }
+    })
+    .then((response) => console.warn(response))
```

Comment:
gotta a response ... but it's the page, no clue to understant why Neomad don't understant that I give a body with data

Reply:
@vinyll same result with postman, so I guess I made a mistake in Neomad API ...